### PR TITLE
Position, rotation and scale now update when edit has finished.

### PR DESCRIPTION
**Pull request by int-Frank**
[AB#1613](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/1613)

### CODE REVIEW

File: src/scene/vcPOI.cpp [hunk 1]
```diff
@@ -838,8 +838,10 @@ void vcPOI::HandleImGui(vcState *pProgramState, size_t *pItemID)
 
     if (m_line.selectedPoint != -1)
     {
-      if (ImGui::InputScalarN(udTempStr("%s##POIPointPos%zu", vcString::Get("scenePOIPointPosition"), *pItemID), ImGuiDataType_Double, &m_line.pPoints[m_line.selectedPoint].x, 3))
-        vcProject_UpdateNodeGeometryFromCartesian(m_pProject, m_pNode, pProgramState->geozone, m_line.closed ? vdkPGT_Polygon : vdkPGT_LineString, m_line.pPoints, m_line.numPoints);
+      ImGui::InputScalarN(udTempStr("%s##POIPointPos%zu", vcString::Get("scenePOIPointPosition"), *pItemID), ImGuiDataType_Double, &m_line.pPoints[m_line.selectedPoint].x, 3);
+
+        if (ImGui::IsItemDeactivatedAfterEdit())
+          vcProject_UpdateNodeGeometryFromCartesian(m_pProject, m_pNode, pProgramState->geozone, m_line.closed ? vdkPGT_Polygon : vdkPGT_LineString, m_line.pPoints, m_line.numPoints);
```
Comment (OsiJr) [inline src/scene/vcPOI.cpp:844]: Indentation seems weird here

File: src/scene/vcPolyModelNode.cpp [hunk 2]
```diff
@@ -151,18 +151,22 @@ void vcPolyModelNode::HandleImGui(vcState *pProgramState, size_t *pItemID)
...
+
+    if (ImGui::IsItemDeactivatedAfterEdit())
     {
       repackMatrix = true;
       orientation = udDoubleQuat::create(UD_DEG2RAD(eulerRotation));
     }
 
-    if (ImGui::InputScalarN(vcString::Get("sceneModelScale"), ImGuiDataType_Double, &scale.x, 3))
-      repackMatrix = true;
+    ImGui::InputScalarN(vcString::Get("sceneModelScale"), ImGuiDataType_Double, &scale.x, 1);
```
Comment (OsiJr) [inline src/scene/vcPolyModelNode.cpp:168]: This doesn't have the deactivated code at all and poly models support 3 axis scaling

File: src/scene/vcPOI.cpp (hunk 1 above)
Comment (int-Frank) [reply src/scene/vcPOI.cpp]: Fixed

File: src/scene/vcPolyModelNode.cpp (hunk 2 above)
Comment (int-Frank) [reply src/scene/vcPolyModelNode.cpp]: Fixed

**Pull request merged**